DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -108,6 +108,11 @@ ISessionDataService sessionDataService
 
         public (AdminUser? adminUser, List<DelegateUser> delegateUsers) GetUsersByEmailAddress(string emailAddress)
         {
+            if (string.IsNullOrWhiteSpace(emailAddress))
+            {
+                return (null, new List<DelegateUser>());
+            }

```

Comment:
Does this impact anywhere else that this method is used?

Additionally, we should probably make the `emailAddress` param nullable as we do handle that case here.

Reply:
The only potential issue I could find is that `NewEmailAddressIsValid` will no longer report a clash if you try to save an empty email, but empty emails should already have been filtered out by form validation so it's meaningless in practice.